packages/kbn-expandable-flyout/src/components/preview_section.tsx

```diff
@@ -95,27 +136,30 @@ export const PreviewSection: React.FC<PreviewSectionProps> = ({
           opacity: 0.5;
         `}
       />
-      <div
+      <EuiSplitPanel.Outer
         css={css`
+          margin: ${euiTheme.size.xs};
+          height: 99%;
           position: absolute;
           top: 0;
           bottom: 0;
           right: 0;
           left: ${left};
           z-index: 1000;
         `}
+        className="eui-yScroll"
+        data-test-subj={PREVIEW_SECTION}
       >
-        <EuiPanel
-          css={css`
-            margin: ${euiTheme.size.xs};
-            height: 100%;
-          `}
-          data-test-subj={PREVIEW_SECTION}
-        >
+        <EuiSplitPanel.Inner grow={false} color={banner?.backgroundColor} paddingSize="none">
```

Comment:
very small improvement here, you can use `isPreviewBanner` to avoid rendering the dom elements if the user doesn't pass anything

```typescript
{isPreviewBanner(banner) && (
  <EuiSplitPanel.Inner grow={false} color={banner.backgroundColor} paddingSize="none">
    <EuiText textAlign="center" color={banner.textColor} size="s">
      {banner.title}
    </EuiText>
  </EuiSplitPanel.Inner>
)}
```